<launch>
	
	<rosparam command="load" file="$(find jaco_learning)/config/demo_recorder.yaml" />
	
	<!-- Node for demonstration recording on Jaco arm. -->
	<node name="demo_recorder" pkg="jaco_learning" type="demo_recorder.py" output="screen"/>

</launch>

